<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsWidget</class>
 <widget class="QWidget" name="settingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="settingsLayout">
   <item>
    <widget class="QLabel" name="gameLabel">
     <property name="text">
      <string>Select your GTA:SA install directory:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="gameWidget" native="true">
     <layout class="QHBoxLayout" name="gameLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="gamePathDisplay">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gameBrowseButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="settingsSpacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="outputLabel">
     <property name="text">
      <string>Choose where to save the ColAndreas file:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="outputWidget" native="true">
     <layout class="QHBoxLayout" name="outputLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="outputPathDisplay">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outputBrowseButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="settingsSpacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="sampObjectsBox">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Include SA-MP objects.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="customObjectsBox">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Include objects from CUSTOM.img</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="settingsSpacer3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
